ui/src/store/resource.test.ts

```diff
@@ -607,4 +609,43 @@ describe('Store functions', () => {
       }
     );
   });
+
+  it('makes sure to add resources with same name but from different catalog', (done) => {
+    const store = ResourceStore.create(
+      {},
+      {
+        api,
+        categories: CategoryStore.create({}, { api })
+      }
+    );
+
+    expect(store.isLoading).toBe(true);
+
+    when(
+      () => !store.isLoading,
+      () => {
+        const item = Resource.create({
+          id: 44,
+          name: 'golang-build',
+          uniqueKey: 'tekton-hub/Task/golang-build',
+          catalog: 2,
+          kind: 'Task',
+          latestVersion: 47,
+          displayVersion: 47,
+          tags: [1],
```

Comment:
instead of creating a resource here shouldn't we update the `resource.json` with a new task having different catalog and test the left pane as well?

Reply:
Adding a resource in `resource.json` will change lot of test cases snapshot and also we won't be able to add a test to check if a resource is added from a different catalog

Reply:
ohh...okay